app/ide-desktop/lib/dashboard/src/utilities/newtype.ts

```diff
@@ -5,7 +5,7 @@
 // ===============
 
 /** An interface specifying the variant of a newtype. */
-interface NewtypeVariant<TypeName extends string> {
+export interface NewtypeVariant<TypeName extends string> {
```

Comment:
do these need to be `export`ed?

Reply:
Yes, because otherwise ts can't access them and show an error https://stackoverflow.com/questions/43900035/ts4023-exported-variable-x-has-or-is-using-name-y-from-external-module-but

Reply:
i guess this is one usecase for using `type`s :skull: